### PR TITLE
Topic 5 improvement tab

### PR DESCRIPTION
**Pull request by audstav**
@seanlip @DubeySandeep PTAL

Changes on this site:
https://oppia-user-guide.readthedocs.io/en/latest/index.html

Note that for the 'Suggestions' task, there was a bug identified, so this section will have to be updated when the bug is fixed.

### CODE REVIEW

**Comment by DubeySandeep** [comment]
@audstav, Can you please resolve the merge conflicts in this PR? (You can find "[how to resolve merge conflicts over](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me)".)

I see lots of duplicate images as well in this PR, not sure why can you please check?

> Note that for the 'Suggestions' task, there was a bug identified, so this section will have to be updated when the bug is fixed.

Can you please add a link to the bug? :)

**Comment by audstav** [comment]
@DubeySandeep Regarding the duplicate images, I don't see any duplicates? There were 26 new ones added in this PR which is what I see in the 'Files changed' tab.

Here is the fix Brian sent out - [#7719](https://github.com/oppia/oppia/pull/7719)

**Review by brianrodri — CHANGES_REQUESTED** [review]
This looks fantastic! Please also add notes explaining that creators can revisit the tasks they resolve by unticking the "Only Show Open Tasks" checkbox at the top (playthroughs are exceptional however, they get permanently deleted).

File: docs/lesson/improvements.rst [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+.. |improvements| image:: /images/improvements_tab.png
+                  :alt: Improvements button
+                  :scale: 30 %
+
+.. _feedback:
+
+Feedback on a card
+-------------------
+
+Learners have the option to provide feedback on each card as they playthrough an exploration, as well as at the end when they are asked to review the exploration. If they have provided any feedback, you will see a task similar to the image below.
```
Comment (brianrodri) [inline docs/lesson/improvements.rst]: missing space: s/playthrough/play through

File: docs/lesson/improvements.rst [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+   :alt: Feedback task
+   :scale: 40 %
+
+   *Fig. 1*
+
+1. To change the status of the feedback or write a message in response to the feedback for your record, click on the |reviewthread| button.
+2. Click |sendandclose| when done.
+
+.. caution:: 
+   Leave the status at *Open* if you are still working on the task. Only use the other options when you are ready to remove the task from the Improvements screen.
```
Comment (brianrodri) [inline docs/lesson/improvements.rst]: Let's say "hidden" instead, remove implies (to me) that they are gone forever. Resolved tasks can be viewed again by unticking the "only show open tasks" checkbox at the top of the improvements tab page.

File: docs/lesson/improvements.rst [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+Learners also have the option to suggest an *edit* on one or more cards, e.g., a change in the content, formatting etc. This comes through as a *Suggestion* task in the Improvements screen.
+
+.. figure:: /images/suggestion_task.png
+   :alt: Suggestion task
+   :scale: 40 %
+
+   *Fig. 3*
+
+1. Click the |reviewthread| button and then the |reviewsuggestion| button to action this suggestion.
```
Comment (brianrodri) [inline docs/lesson/improvements.rst]: "to take action on this suggestion"

File: docs/lesson/improvements.rst [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+2. To delete the **Answer details** task, checkmark the **Select** box and click on the |deleteitems| box.
+
+.. |deleteitems| image:: /images/delete_selected_items.png
+                 :alt: Delete Selected items box
+                 :scale: 40 %
+
+
+.. admonition:: An example on improving an exploration
+
+   Let's say you asked the question: **Which is greater: 0.834 or 0.2?** and requested an explanation from the learner for their answer. The learner selected the right answer but explained they chose it because it was a longer number. This is obviously a misconception, so you may then decide to include additional cards beforehand or :ref:`branch out <addbranch>` to focus on place value or the decimal-fraction link.
```
Comment (brianrodri) [inline docs/lesson/improvements.rst]: Please add comma: "The learner selected the right answer, but explained they..."

**Review by brianrodri — CHANGES_REQUESTED** [review]

**Comment by DubeySandeep** [comment]
> Regarding the duplicate images, I don't see any duplicates? There were 26 new ones added in this PR which is what I see in the 'Files changed' tab.

I think before you resolved merge conflicts it was showing duplicate images from your last PR, now It's fine! :)

**Review by DubeySandeep — COMMENTED** [review]
@audstav, This looks great to me, I'm waiting for @anubhavsinha98's review once he approves I'll go ahead and merge this PR!

(In the meantime you can address @brianrodri's review comment!)

File: docs/lesson/improvements.rst (hunk 1 above)
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

File: docs/lesson/improvements.rst (hunk 2 above)
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

File: docs/lesson/improvements.rst (hunk 3 above)
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

File: docs/lesson/improvements.rst (hunk 4 above)
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

**Comment by audstav** [comment]
@DubeySandeep 
I've incorporated suggested changes and updated the video as well.

**Comment by DubeySandeep** [comment]
@audstav Thanks for the changes, @brianrodri can you please take a final look?

File: docs/lesson/improvements.rst [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+                         :scale: 40 %
+
+.. figure:: /images/answer_details_review.png
+   :alt: Review answer details
+   :scale: 40 %
+
+   *Fig. 9*
+
+
+2. To delete the **Answer details** task, checkmark the **Select** box and click on the |deleteitems| box.
```
Comment (anubhavsinha98) [inline docs/lesson/improvements.rst]: To delete any **Answer details** received, checkmark the **Select** box and click on......
You can also delete multiple answer details at a time.

**Review by anubhavsinha98 — CHANGES_REQUESTED** [review]
Hi @audstav, looks good. Just one minor change!
Thanks :)

File: docs/lesson/improvements.rst (hunk 5 above)
Comment (audstav) [reply docs/lesson/improvements.rst]: Done

**Comment by audstav** [comment]
@DubeySandeep Anubhav's changes now incorporated!

**Review by anubhavsinha98 — APPROVED** [review]
LGTM :)

**Review by DubeySandeep — APPROVED** [review]

**Comment by DubeySandeep** [comment]
@brianrodri, Can you please take a final look at the changes made as per your comments?

**Comment by DubeySandeep** [comment]
I'm going to merge this PR as per our email thread and @brianrodri can review the PR once he is back and @audstav can create a follow-up PR if required!

**Pull request merged**